src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/InternalIPAddressCollection.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+
+namespace System.Net.NetworkInformation
+{
+    public class InternalIPAddressCollection : IPAddressCollection
```

Comment:
Makes sense. This appeared because we wanted to keep things in one place (regardless the actual namespace): we moved IPAddress and IPAddressCollection into Primitives.

When splitting things up, please add the correct suffix of `.Windows.cs` or `.Linux.cs`.